### DISCUSSION TITLE
Need to get the max lengh of a SharePoint text field

### DISCUSSION THREAD

**Discussion by sabrinadeveloper2023** [pnp/pnpcore#1174] [General]
Hi,

How I can check the length of a text field using pnp core?

Thanks

**Comment by jansenbe** [comment]
@sabrinadeveloper2023 : text fields default to 255 characters unless specified otherwise at creation time. When you read a field you can read the `SchemaXml` property which contains the field schema, in there you'll for example find `MaxLength="50"` .

```Xml
<Field DisplayName="MaxLenghtField" Format="Dropdown" IsModern="TRUE" MaxLength="50" Name="MaxLenghtField" Title="MaxLenghtField" Type="Text" ID="{70be9b6a-7946-4015-98ff-3dd2dec23c8a}" SourceID="{91bb170e-0ef0-4062-86f9-e5dbe8d047bd}" StaticName="MaxLenghtField" ColName="nvarchar7" RowOrdinal="0" />
```
